### ISSUE TITLE
 Mask-RCNN object detection and segmentation: Compatibility issues with Tensorflow and keras

### ISSUE THREAD

**Issue by kavi0786** [TannerGilbert/MaskRCNN-Object-Detection-and-Segmentation#4]
Hello sir, i have executed the Mask-RCNN object detection and segmentation in google colab... But , the follwoing errors present every time...

Attribute Error: module 'tensorflow._api.v1.compat.v2' has no attribute '__internal__'.

Attribute Error: module 'keras.utils.generic_utils' has no attribute 'populate_dict_with_module_objects'..

Not only these errors,,  Always problem with tensorflow and keras comaptabilty issues  in google colab...

can you give any solution ? Thanks in advance...

**Comment by TannerGilbert** [comment]
What Tensorflow and Keras version are you using?

**Comment by kavi0786** [comment]
Sir, in google colab, I used tensorflow==1.14.0
Keras===2.3.0..

**Comment by TannerGilbert** [comment]
It seems to work with Tensorflow 1.15.2 and Keras 2.3.1. I'll update the notebooks to use this version.

**Comment by TannerGilbert** [comment]
Actually it wasn't quite that simple. I aso ran into some other issues with that versions that I had to solve. I updated all the notebooks. Please take a look at them for more detail.
